zombie_dice/ai.go

```diff
@@ -0,0 +1,24 @@
+package zombie_dice
+
+import (
+	"github.com/Akavall/GoGamesProject/dice"
+)
+
+func GreedyAI(shots int) int {
```

Comment:
It mostly just generalizes the AI, so if we want to keep programmatically adding new AIs the interface provides a sort of contract of what methods we expect to be implemented in each new AI. 

E.g. no matter what different AI there is implemented for Zombie Dice we would expect there to always be a "shouldKeepGoing()" method that we can call.